src/bin/am/server/assets.rs

```diff
@@ -0,0 +1,18 @@
+use crate::server::util::proxy_handler;
+use axum::body::Body;
+use axum::response::IntoResponse;
+use http::header::CONNECTION;
+use url::Url;
+
+pub async fn handler(mut req: http::Request<Body>, upstream_base: Url) -> impl IntoResponse {
```

Comment:
Think we can just add this to the explorer mod instead of creating a new one.

Reply:
Sounds good. I was trying to follow the convention but indeed it is a pretty lightweight function indeed.